modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,100 @@
+For example:
+
+Hub cluster resources::
+Available hub cluster host resources (CPU, memory, storage) are an important factor in determining how many clusters the hub cluster can manage.
+The more resources allocated to the hub cluster, the more managed clusters it can accommodate.
+
+Hub cluster storage::
+The hub cluster host storage IOPS rating and whether the hub cluster hosts use NVMe storage can affect hub cluster performance and the number of clusters it can manage.
+
+Network bandwidth and latency::
+Slow or high-latency network connections between the hub cluster and managed clusters can impact how the hub cluster manages multiple clusters.
+
+Managed cluster size and complexity::
+The size and complexity of the managed clusters also affects the capacity of the hub cluster.
+Larger managed clusters with more nodes, namespaces, and resources require additional processing and management overhead.
+Similarly, clusters with complex configurations such as the RAN DU profile or diverse workloads can require more resources from the hub cluster.
+
+Number of managed policies::
+The number of policies managed by the hub cluster scaled over the number of managed clusters bound to those policies is an important factor that determines how many clusters can be managed.
+
+Monitoring and management workloads::
+{rh-rhacm} continuously monitors and manages the managed clusters.
+The number and complexity of monitoring and management workloads running on the hub cluster can affect its capacity.
+Intensive monitoring or frequent reconciliation operations can require additional resources, potentially limiting the number of manageable clusters.
+
+{rh-rhacm} version and configuration::
+Different versions of {rh-rhacm} can have varying performance characteristics and resource requirements.
+Additionally, the configuration settings of {rh-rhacm}, such as the number of concurrent reconciliations or the frequency of health checks, can affect the managed cluster capacity of the hub cluster.
+
+Use the following representative configuration and network specifications to develop your own Hub cluster and network specifications.
+
+[IMPORTANT]
+====
+The following guidelines are based on internal lab benchmark testing only and do not represent a complete real-world host specification.
+====
+
+.Representative three-node hub cluster machine specifications
+[cols=2*, width="90%", options="header"]
+|====
+|Requirement
+|Description
+
+|{product-title} version
+|version 4.13
+
+|{rh-rhacm} version
+|version 2.7
+
+|{cgu-operator-first}
+|version 4.13
+
+|Server hardware
+|3 x Dell PowerEdge R650 rack servers
+
+|NVMe hard disks
+a|* 50 GB disk for `/var/lib/etcd`
+* 2.9 TB disk for `/var/lib/containers`
+
+|SSD hard disks
+a|* 1 SSD split into 15 200GB thin-provisioned logical volumes provisioned as `PV` CRs
+* 1 SSD serving as an extra large `PV` resource
+
+|Number of applied DU profile policies
+|5 (6 if you include a validation policy)
```

Comment:
@akrzos Look OK to you?

Reply:
No AIUI, it should be just 5 policies total. This is because there are 3 policies plus a site policy and a du validation policy.  In the scale lab tests I rename two parts of the PGT to match this same 5 policy count.